roles/st2/tasks/2.version.yml

```diff
@@ -32,6 +32,12 @@
   when: st2_revision == 'current'
```

Comment:
This line and entire block below `name: version | Set which StackStorm revision to install manually` is uneeded.

Instead of that, we'll need to remove this line to pickup manual revision. After that both will work:
https://downloads.stackstorm.net/releases/st2/1.2dev/debs/current/VERSION.txt
`74`

https://downloads.stackstorm.net/releases/st2/1.2dev/debs/74/VERSION.txt
`74`

As a bonus by using this approach, we'll get validation if manual revision exists by fetching that url.